### PR TITLE
feat/new-wpt-arguments

### PR DESCRIPTION
**Pull request by FeliceGeracitano**
Add User agent string in WPT endpoint to enable synthetic loading

### CODE REVIEW

File: packages/inspector-wpt/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "inspector-wpt",
-  "version": "1.0.0",
+  "version": "0.0.1",
```
Comment (FeliceGeracitano) [inline packages/inspector-wpt/package.json]: align package to docker image https://cloud.docker.com/u/felicegeracitano/repository/docker/felicegeracitano/inspector-wpt

File: packages/inspector-wpt/types/wpt.d.ts [hunk 2]
```diff
@@ -117,3 +117,11 @@ export interface IDBPayload {
   content_other_compressed: number;
   content_other_uncompressed: number;
 }
+
+export interface IAudit {
+  url: string;
+  saveInDB?: boolean;
+  mobile?: '1' | '0';
+  fvonly?: '1' | '0';
+  uastring?: string;
```
Comment (FeliceGeracitano) [inline packages/inspector-wpt/types/wpt.d.ts]: new user agent string param

File: packages/inspector-wpt/src/audit-route.ts [hunk 3]
```diff
@@ -1,19 +1,19 @@
 import * as express from 'express';
-import utils from './utils';
+import { IAudit } from 'wpt';
```
Comment (rbelling) [inline packages/inspector-wpt/src/audit-route.ts]: I don't see `wpt` listed in package.json dependencies, is that part of a separate PR?

File: packages/inspector-wpt/src/utils.ts [hunk 4]
```diff
@@ -44,10 +44,16 @@ const loopUntillGetRespose = async url => {
   });
 };
 
-const audit = async (url: string, mobile: string, fvonly: string): Promise<any> => {
+const audit = async (
+  url: string,
+  mobile: string,
+  fvonly: string,
```
Comment (rbelling) [inline packages/inspector-wpt/src/utils.ts]: fv... first view?

File: packages/inspector-wpt/src/utils.ts [hunk 5]
```diff
@@ -44,10 +44,16 @@ const loopUntillGetRespose = async url => {
...
+  uastring?: string
+): Promise<any> => {
   try {
     console.log(`Getting data for ${url}`);
-    const runTestUrl = `${WPT_SERVER_URL}/runtest.php?f=json&url=${url}&mobile=${mobile}&fvonly=${fvonly}`;
+    const agentstring = uastring && `&uastring=${uastring}`;
+    const runTestUrl = `${WPT_SERVER_URL}/runtest.php?f=json&url=${url}&mobile=${mobile}&fvonly=${fvonly}${agentstring}`;
```
Comment (rbelling) [inline packages/inspector-wpt/src/utils.ts]: agentstring is of type boolean, right? so the runTestUrl would end with true/false. Is that intended?

File: packages/inspector-wpt/src/audit-route.ts (hunk 3 above)
Comment (rbelling) [reply packages/inspector-wpt/src/audit-route.ts]: I see it's an interface... sorry didn't realize this was typescript

File: packages/inspector-wpt/src/utils.ts (hunk 4 above)
Comment (rbelling) [reply packages/inspector-wpt/src/utils.ts]: might be worth adding a reference to https://github.com/WPO-Foundation/webpagetest/blob/5938639c56242e8dd6d0fde11a434f965820e0f6/www/index.php#L357 so we can easily see where / how those parameters are used in WPT :)

Comment (FeliceGeracitano) [reply packages/inspector-wpt/src/utils.ts]: ok added in the td file: `packages/inspector-wpt/types/wpt.d.ts`

**Pull request merged**